### ISSUE TITLE
Crash while streaming music in TCP.onStreamRead()

### ISSUE THREAD

**Issue by amatayamada** [ytmdesktop/ytmdesktop#126]
Windows 10 version 1903 (OS build 18362.657), ytmdesktop 1.9.0

I am streaming music without problems for 2-3 minutes behind a HTTP proxy, then this error appears, stopping the music. Closing the error window resumes the streaming. Possible duplicate of #112 or #87, but I opened a new issue since the error message is different. 

If there's a more detailed error log I could provide, happy to help.

![error image](https://user-images.githubusercontent.com/27951815/77744982-cbdde900-701a-11ea-8626-99ba64acdaf8.png)

```
[Window Title]
Error

[Main Instruction]
A JavaScript error occurred in the main process

[Content]
Uncaught Exception:
TypeError: Error processing argument at index 0, conversion failure from undefined
    at C:\Users\User\AppData\Local\Programs\youtube-music-desktop-app\resources\app.asar\providers\trayProvider.js:69:31
    at ClientRequest.<anonymous> (C:\Users\User\AppData\Local\Programs\youtube-music-desktop-app\resources\app.asar\node_modules\base64-img\base64-img.js:81:21)
    at ClientRequest.emit (events.js:203:13)
    at Socket.socketOnData (_http_client.js:456:9)
    at Socket.emit (events.js:203:13)
    at addChunk (_stream_readable.js:295:12)
    at readableAddChunk (_stream_readable.js:276:11)
    at Socket.Readable.push (_stream_readable.js:210:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:166:17)

[OK]
```

**Comment by adlerluiz** [comment]
the error happens when the system is unable to download the album image for notification

**Comment by rickinbsb** [comment]
Fixed on 1.10
